README.md

```diff
@@ -7,69 +7,70 @@
 
 This repository list of proposals for the Strimzi project. A template for new proposals can be found [here](./000-template.md).
 
-|  #  | Title                                                                 |
```

Comment:
Why did you change this file, by removing a space in the `#` column? I would leave that for when 100th proposal will come :-)

Reply:
Sorry, that was automatic table alignment done by the Idea's MD editor. Should I revert?

Reply:
I would yes, because it's not related to the PR.

Reply:
ok